server/emailTemplates/verification.js

```diff
@@ -0,0 +1,40 @@
+import Mailgen from 'mailgen';
+import { tokenGenerator, getFullUrl, getBaseUrl } from '<helpers>/utils';
+
+const APP_NAME = 'Author\'s Haven - King-Kong Devs';
+
+const getMailGenerator = link => new Mailgen({
+  theme: 'default',
+  product: {
+    name: APP_NAME,
+    link
+  }
+});
+
+const emailTemplate = link => ({
```

Comment:
I think some of this logic like getMailGenerator function, APP_NAME etc can be extracted to a `config` or an `index` file to be used by other emailTemplates